gui/src/app/view/settings.rs

```diff
@@ -479,8 +481,17 @@ pub fn rescan<'a>(
                                 .align_items(Alignment::Center)
                                 .spacing(10),
                         )
+                        .push_maybe(if invalid_date {
+                            Some(
+                                p1_regular("Date must be valid and after the 03 Jan 2009")
+                                    .style(color::RED),
```

Comment:
I'm afraid it could confuse a user who gets this error while they input a date after Jan 3rd 2009. How about simply
```suggestion
                                p1_regular("Invalid date.")
                                    .style(color::RED),
```

Or to get fancy:
```suggestion
                                p1_regular("Provided date is either invalid or anterior to the genesis block's timestamp.")
                                    .style(color::RED),
```